index.js

```diff
@@ -59,7 +59,7 @@ module.exports = function(fn) {
 
     res.end = function(chunk, encoding, cb) {
       debug('end called');
-      var args = Array.prototype.slice.call(arguments, 1);
+      var args = Array.prototype.slice.call(arguments);
```

Comment:
I'm not sure I understand what you mean. :-)

There were no tests catching this, but I found it as soon as I started using it in a non-streaming context. If you use it to only intercept responses from a handler that streams, you might never notice this (fx. if it's only things served by the default static handler).